module-4/M4/server.js

```diff
@@ -0,0 +1,31 @@
+import http from 'http';
+import fs from 'fs';
+// import _ from 'lodash';
+import express from 'express';
+import mongoose from 'mongoose';
+import blogsRouter from "./routes/blogsRouter.js";
+
+
+const app = express();
+
+const dbURI = 'mongodb+srv://yimingxu96:UC0tdVisA3LkRhml@m4.sdxiz04.mongodb.net/m4-assignment?retryWrites=true&w=majority&appName=M4'
```

Comment:
Need to remove this.
Even if it's "old code" in the modules 4 section, hackers can find this and use it to use up your MongoDB account, possibly causing bills in your name